tests/unit/amplitude/expected_recipe.yml

```diff
@@ -1264,28 +1264,6 @@ ParticleList:
       CParity: 1
       GParity: 1
       IsoSpin: 0
-  f2(1270):
-    PID: 225
-    Mass: 1.2755
-    Width: 0.1867
-    QuantumNumbers:
-      Spin: 2
-      Charge: 0
-      Parity: 1
-      CParity: 1
-      GParity: 1
-      IsoSpin: 0
-  f2(1950):
-    PID: 9050225
-    Mass: 1.936
-    Width: 0.464
-    QuantumNumbers:
-      Spin: 2
-      Charge: 0
-      Parity: 1
-      CParity: 1
-      GParity: 1
-      IsoSpin: 0
```

Comment:
Why was this removed?

Reply:
Because I set the allowed intermediate states to `"f0"` as to speed it up. The test only checks whether the recipe file looks as expected.

Reply:
Yeah I thought so, but then I dont understand why only this part is removed and not the amplitude parts as well?

Reply:
That is... a good point :thinking:

Reply:
https://github.com/ComPWA/expertsystem/blob/1d682f52b530d1a75a62a189e642f3ed0261847f/tests/unit/amplitude/test_yaml_helicity.py#L146-L148
Seems that the test is just incomplete. But now I remember this test sort of stalled while #22 is in the making. We didn't want to put too much effort into this temporary `_yaml_adapter` thingy until then.

Reply:
I think the problem is that this test is not really a unit test. And this change is just a workaround to make it faster

Reply:
That's true. And that is in turn related to the fact that we have no representation of an amplitude module. For now all that test does is write the file out and see if that works (plus a check on the format of the `ParticleList` and `Kinematics` section.

Reply:
181d98ebdc87497857f61c34062d246d671260e5
Well, super dirty hack, but I'm not sure how to make this into a proper unit test right now